msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -551,10 +555,12 @@ public void checkIntentFilterAddedToAppManifestForBrokerFlow() throws MsalClient
             return;
         }
 
-        if (!isBrokerRedirectUri()) {
+        if (!isBrokerRedirectUri(mRedirectUri, mAppContext.getPackageName())) {
             // This means that the app is still using the legacy local-only MSAL Redirect uri (already removed from the new portal).
             // If this is the case, we can assume that the user doesn't need Broker support.
-            Logger.info(TAG, "The app is still using legacy MSAL redirect uri. Switch to MSAL local auth.");
+            Logger.info(TAG, "The app is still using legacy MSAL redirect uri. Switch to MSAL local auth."
```

Comment:
See that this was an info message previously.  We may want to warn here instead.